core/templates/dev/head/pages/admin/admin.mainpage.html

```diff
@@ -70,6 +70,23 @@
 
     {% include 'pages/footer_js_libs.html' %}
 
+    <script src="/templates/dev/head/pages/admin/admin.module.js"></script>
```

Comment:
Add comments specifying that the module is for admin page.

Reply:
done